python_modules/dagster-graphql/dagster_graphql_tests/graphql/graphql_context_test_suite.py

```diff
@@ -377,6 +423,8 @@ class Marks:
     multi_location = pytest.mark.multi_location
     managed_grpc_env = pytest.mark.managed_grpc_env
     deployed_grpc_env = pytest.mark.deployed_grpc_env
+    code_server_cli_grpc_env = pytest.mark.code_server_cli_grpc_env
```

Comment:
im not sure the best place to do it -but we should write down exactly what the difference is between the three conditions here since i dont think it would be easy for someone to re-derive the underlying assumed behavior differences that are the reason we test each

Reply:
The labels of the marks in the pytest.ini do a decent summary i think, i'll add it to the implementation as well